src/commonMain/kotlin/arrow.exact/ExactError.kt

```diff
@@ -0,0 +1,3 @@
+package arrow.exact
+
+class ExactError(val message: String)
```

Comment:
We can make this class `open` so people can define custom exact errros that can be handled easily programmatically.